### PR TITLE
Build scripts

### PR DESCRIPTION
**Pull request by jean-edouard**

### CODE REVIEW

File: build-scripts/oe/build.sh [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+REPO_PROD_CACERT="/home/build/certs/prod-cacert.pem"
+REPO_DEV_CACERT="/home/build/certs/dev-cacert.pem"
+REPO_DEV_SIGNING_CERT="/home/build/certs/dev-cacert.pem"
+REPO_DEV_SIGNING_KEY="/home/build/certs/dev-cakey.pem"
+EOF
+
+./do_build.sh -s setupoe | tee setupoe.log
+
+# now remove setupoe from STEPS:
+sed -i 's/^STEPS="setupoe,/STEPS="/' ./do_build.sh
```
Comment (rossphilipson) [inline build-scripts/oe/build.sh]: Please remove hackery

File: build-scripts/oe/build.sh [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+git fetch
+git checkout jethro-merge
+git checkout master
+git config user.email "build@localhost"
+git config user.name "automated build"
+git merge --no-edit jethro-merge
+cd -
+
+# Apply Eric's temporary workaround:
+sed -i 's/^LOCALE_GENERATION_WITH_CROSS-LOCALEDEF = "1"/LOCALE_GENERATION_WITH_CROSS-LOCALEDEF = "0"/' ./build/repos/openembedded-core/meta/recipes-core/glibc/glibc-locale.inc
```
Comment (rossphilipson) [inline build-scripts/oe/build.sh]: Please remove hackery

File: build-scripts/centos/build.sh [hunk 3]
```diff
@@ -0,0 +1,83 @@
+#!/bin/sh
```
Comment (rossphilipson) [inline build-scripts/centos/build.sh]: Please add GPL header and copyrights

File: build-scripts/debian/build.sh [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+SBUILD="sbuild --purge-deps=never"
+
+# Build
+mkdir $BUILD_DIR
+cd $BUILD_DIR
+git clone -b lxc https://github.com/jean-edouard/pv-linux-drivers.git
+git clone -b build-scripts https://github.com/jean-edouard/v4v.git
+git clone -b build-scripts https://github.com/jean-edouard/xctools.git
+cp -r v4v/v4v/linux v4v/libv4v/src/
```
Comment (rossphilipson) [inline build-scripts/debian/build.sh]: @jean-edouard do you want your repos here - that seems wrong.

File: build-scripts/oe/build.sh [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+# now remove setupoe from STEPS:
+sed -i 's/^STEPS="setupoe,/STEPS="/' ./do_build.sh
+
+# get the right jethro xenclient-oe repo
+# and again merge into master to use it
+cd build/repos/xenclient-oe
+git remote rm origin || echo ok no prior origin
+git remote add origin "https://github.com/aikidokatech/xenclient-oe.git"
+git fetch
```
Comment (rossphilipson) [inline build-scripts/oe/build.sh]: No longer correct.

**Comment by jean-edouard** [comment]
All fixed now.
This PR depends on:
https://github.com/OpenXT/pv-linux-drivers/pull/30
https://github.com/OpenXT/xctools/pull/21
https://github.com/OpenXT/v4v/pull/8

**Pull request merged**